message-me/app/javascript/packs/application.js

```diff
@@ -12,9 +12,19 @@ import '@doabit/semantic-ui-sass'
 Rails.start()
 Turbolinks.start()
 ActiveStorage.start()
+
+window.scroll_bottom = function(id) {
+  const element = $(`#${id}`)
+  const scrollHeight = element[0].scrollHeight;
+  if (id.length > 0){
+    element.scrollTop(scrollHeight);
+  }
+};
```

Comment:
```suggestion
  const element = $(`#${id}`)
  if (element.length > 0){
    const scrollHeight = element[0].scrollHeight;
    element.scrollTop(scrollHeight);
  }
};
```
Shouldn't it be this?